desktop/frontend/src/app.css

```
@@ -113,5 +113,507 @@ button {
+    border-color: #ff6b6b;
+    color: #ff6b6b;
+}
+
+.action-btn--red:hover {
+    background: rgba(255, 107, 107, 0.15);
+    box-shadow: 0 0 12px rgba(255, 107, 107, 0.3);
+}
   to { transform: translateY(20px); opacity: 0; }
-}
\ No newline at end of file
+}
```

Stylelint (17.6.0)

[error] Unexpected } (CssSyntaxError)

(CssSyntaxError)